### PR TITLE
Overpass fine tuning

### PR DESCRIPTION
**Pull request by RoPP**

### CODE REVIEW

File: roles/overpass-api/tasks/munin.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+- name: install munin plugins
+  template:
+    src: templates/munin/osm_db_lag
+    dest: /etc/munin/plugins
+    mode: 0755
+
+- name: reload munin-node
+  systemd:
+    name: munin-node
+    state: restarted
```
Comment (jocelynj) [inline roles/overpass-api/tasks/munin.yml]: For information, you can use `notify: reload munin-node`, with a file in handlers/munin-node.yml containing something like this:
```
- name: reload munin-node
  service: name=munin-node state=reloaded
```
(should be easily adapted with your systemd syntax)

Comment (RoPP) [reply roles/overpass-api/tasks/munin.yml]: Yes true, same occurs for nginx. Will correct that.

File: roles/overpass-api/tasks/set_up_database.yml [hunk 2]
```diff
@@ -30,3 +31,4 @@
     enabled: yes
     daemon_reload: yes
     no_block: yes
+  when: service.changed
```
Comment (jocelynj) [inline roles/overpass-api/tasks/set_up_database.yml]: Isn't this automatically handled by ansible ?

Tasks are supposed to not do anything if not necessary.

Comment (RoPP) [reply roles/overpass-api/tasks/set_up_database.yml]: It is, but this service is stop and inhibed after its first run. Ansible see it stopped and then try to restart it.

Comment (jocelynj) [reply roles/overpass-api/tasks/set_up_database.yml]: Ok. It is a strange behaviour, but that's fine.

**Pull request merged**